contrib/perf/get-pr-info.py

```
@@ -54,19 +55,41 @@ def get_prs_from_event_file(event_path: Path) -> List[int]:
+    if workflow_run.get('event') != 'pull_request':
+        return []
 
-    return pr_numbers
+    head_branch = workflow_run.get('head_branch')
+    if not head_branch:
+        return []
+
+    print(f"Searching for PRs by head_branch: {head_branch}", file=sys.stderr)
+
+    # Use gh CLI to search for PRs by head branch
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)